kategory-core/src/main/kotlin/kategory/instances/NumberInstances.kt

```diff
@@ -32,6 +40,14 @@ object DoubleSemigroupInstanceImplicits {
     @JvmStatic fun instance(): Semigroup<Double> = SGDouble
 }
 
+object DoubleOrderInstance : Order<Double> {
```

Comment:
You may have to add these implicit objects also to `NumberInstancesJava`. Unfortunately the implicits lookup system that relies on reflection for the time being sees `kotlin.Double` as `java.lang.Double` and we have a special hack to handle those.

Reply:
nevermind, you already did!